war/src/main/js/keyboard-shortcuts.js

```diff
@@ -0,0 +1,24 @@
+import hotkeys from "hotkeys-js"
+
+window.addEventListener("load", () => {
+  const searchBar = document.querySelector("#search-box")
+  searchBar.placeholder = searchBar.placeholder + ` (${translateKeyboardShortcutForOS("CMD+K")
+    .replace("CMD", "⌘")})`
+
+  hotkeys(translateKeyboardShortcutForOS("CMD+K"), () => {
+    searchBar.focus()
+
+    // Returning false stops the event and prevents default browser events
+    return false
+  })
+})
+
+/**
+ * Translates a given keyboard shortcut, e.g. CMD+K, into an OS neutral version, e.g. CTRL+K
```

Comment:
This doesn't actually describe what this does correctly.

Also, is this intended to be reused elsewhere? Otherwise this is unnecessarily generic.

Reply:
```js
/**
 * Given a keyboard shortcut, e.g. CMD+K, replace any included modifier keys for the user's
 * platform e.g. output will be CMD+K for macOS/iOS, CTRL+K for Windows/Linux
 * @param {string} keyboardShortcut The shortcut to translate
 */
function translateModifierKeysForUsersPlatform(keyboardShortcut) {
  const useCmdKey = navigator.platform.toUpperCase().indexOf("MAC") >= 0 ||
    navigator.platform.toUpperCase() === "IPHONE"
  return keyboardShortcut.replaceAll(/CMD|CTRL/ig, useCmdKey ? "CMD" : "CTRL")
}
```

I've updated it now to be more descriptive, hope that's better. 

For this PR it's limited to translating CMD+K to CTRL+K if you're on a non macOS/iOS platform, but once developers can integrate their plugins it'll be used to translate the shortcuts they use too.